plugin/trino-google-sheets/src/main/java/io/trino/plugin/google/sheets/SheetsClient.java

```diff
@@ -133,8 +150,8 @@ public Optional<SheetsTable> getTable(String tableName)
                 columnNames.add(columnValue);
                 columns.add(new SheetsColumn(columnValue, VarcharType.VARCHAR));
             }
-            List<List<String>> dataValues = values.subList(1, values.size()); // removing header info
-            return Optional.of(new SheetsTable(tableName, columns.build(), dataValues));
+            List<List<String>> dataValues = stringValues.subList(1, stringValues.size()); // removing header info
+            return Optional.of(new SheetsTable(tableName.orElse("_generated"), columns.build(), dataValues));
```

Comment:
Note here in `SheetsTable` the tableName is required, but is completely unused.

Is it worth removing the tableName from `SheetsTable`?

Reply:
I agree with removing the variable.

Reply:
removed.